documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -219,6 +219,8 @@ GRANT SELECT ON V_$DATABASE to c##logminer CONTAINER=ALL;
 GRANT FLASHBACK ANY TABLE TO c##logminer CONTAINER=ALL;
 GRANT LOGMINING TO c##logminer CONTAINER=ALL;
 GRANT LOCK ANY TABLE TO c##logminer CONTAINER=ALL;
+GRANT CREATE TABLE TO c##logminer CONTAINER=ALL;
```

Comment:
Why does it need `CREATE TABLE`?

Reply:
The `LogMiner` implementation creates table `LOG_MINING_AUDIT` if it doesn't exist.  See [SqlUtils.java](https://github.com/debezium/debezium-incubator/blob/master/debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/SqlUtils.java#L43). 

If I interpret the comments from LogMinerHelper#getEndScn where that table is updated, it seems the connector uses this to flush the redo log writer.   I assume we'd prefer not to have this?  If so, I'll can open an issue and review what the impacts are if we remove this from the implementation.

Reply:
Yeah, so let's discuss with @AndreyIg about this, and if it's truly needed, this should be mentioned in the docs very clearly. So far, no connectors write back to the database (apart from Postgres heartbeat action queries), so this needs clear documentation.

Reply:
So after discussing with @AndreyIg, my intepretation of the comments were correct.  Essentially this mechanism is used to trigger Oracle to flush the redo write buffer to disk so that during the next log mining session, we get a full picture of what has happened.  I'll update the documentation in this PR so this is clearer if it isn't already.

Reply:
Ok, thanks, makes sense. Feel free to merge it yourself then.